ui/app/components/app/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js

```diff
@@ -88,29 +73,6 @@ export default class AdvancedTabContent extends Component {
               minimumGasLimit={minimumGasLimit}
             />
           </div>
-          {isEthereumNetwork ? (
-            <div>
-              <div className="advanced-tab__fee-chart__title">
-                {t('liveGasPricePredictions')}
-              </div>
-              {gasEstimatesLoading ? (
-                <Loading />
-              ) : (
-                <GasPriceChart
-                  {...gasChartProps}
-                  updateCustomGasPrice={this.onGasChartUpdate}
```

Comment:
`this.onGasChartUpdate` is now unused

Reply:
That and a couple other things deleted in 860e7108c